zerver/lib/message.py

```diff
@@ -168,7 +168,10 @@ class SendMessageRequest:
     default_bot_user_ids: Set[int]
     service_bot_tuples: List[Tuple[int, int]]
     all_bot_user_ids: Set[int]
+    topic_wildcard_mention_user_ids: Set[int]
     stream_wildcard_mention_user_ids: Set[int]
```

Comment:
Does `topic_wildcard_mention_user_ids` include `topic_wildcard_mention_in_followed_topic_user_ids`? It might be really good to have a comment here making that extremely clear.

Reply:
Comment added.